src/Command/MarkMigrated.php

```diff
@@ -60,7 +60,9 @@ protected function configure()
             ));
         $this->addOption('plugin', 'p', InputArgument::OPTIONAL, 'The plugin the file should be created for')
             ->addOption('connection', 'c', InputArgument::OPTIONAL, 'The datasource connection to use')
-            ->addOption('source', 's', InputArgument::OPTIONAL, 'The folder where migrations are in');
+            ->addOption('source', 's', InputArgument::OPTIONAL, 'The folder where migrations are in')
+            ->addOption('skip-to', 'u', InputArgument::OPTIONAL, 'If present it will mark all versions until the given one')
+            ->addOption('up-to', 't', InputArgument::OPTIONAL, 'If present it will mark all versions including the given one');
```

Comment:
About InputArgument / InputOption, it is a good point. It would be worthwhile to check if switching them to ``InputOption::VALUE_OPTIONAL`` does not break everything. That would make things cleaner and more consistent. But this can be done in a separate PR.

About the naming of the option, I'll give it to you for ``--up-to`` : it is self explanatory. 
But the ``--skip-to`` wording does not reflect what it does. Reading it, I understand "Skip migrations up to this one".
I still think the exclusion of the targeted migration should depend on another flag.
And that ``up-to`` should be ``target`` to keep consistency with the rest of the commands (migrate, rollback). appens